src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala

```diff
@@ -43,6 +43,9 @@ import utils._
 object Enrich {
 
   private val logger = LoggerFactory.getLogger(this.getClass)
+
+  implicit val badRowScioCodec: Coder[BadRow] = Coder.kryo[BadRow]
```

Comment:
Are we sure it will be enough? @dilyand do you remember what our workaround included?

Reply:
If it can compile it should be. When I tried to do it few months ago compiling was always resulting in OOM.

Reply:
IIRC we then hit runtime exception because Beam couldn't serialize `Map`. @dilyand probably remembers how we fixed this.

Reply:
If I remember correctly, the serialization issue only comes to light if we use `BadRow` in a `groupBy`. 

But anyway, the solution was a custom mapper:https://github.com/snowplow-product/gcp-bad-row-aggregator/blob/master/src/main/scala/com/snowplowanalytics/monitoring/googlecloud/badrows/aggregates/DMapper.scala

Reply:
thanks Dilyan!